src/services/auth/auth.tsx

```
@@ -12,7 +12,9 @@ interface MobileData {
   otp: number;
   token: string;
 }
-
+function handleError(error: any) {
+  throw error.response ? error.response.data : new Error("Network Error");
```

it may not be network error all the times?

---

can we throw error as "An unexpected error occurred."